device-types/Cisco/N3K-C3064TQ-10GT.yaml

```diff
@@ -0,0 +1,125 @@
+---
+manufacturer: Cisco
+model: Nexus C3064TQ-10GT
+slug: cisco-n3k-c3064tq-10gt
+part_number: N3K-C3064TQ-10GT
+u_height: 1
+is_full_depth: false
```

Comment:
Hello, thank you for your contribution! In an attempt to have the most accurate and complete data we can, I have a few additional data points we would like you to add if possible.

1. Please add the `airflow` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not
2. Please add the `weight` and associated `weight_unit` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not

If you have any questions, please let us know :)

Reply:
Hi,
1. for the airflow, Front to Rear and Rear to Front are possible ! How can I specify the both ?
Thx

Reply:
If airflow is determined via the PSU, then you can ignore airflow. Please just add `weight` and `weight_unit` to this file